### ISSUE TITLE
Destructable cage 

### ISSUE THREAD

**Issue by MaxeUA** [studio-minus/ppg-bugs-and-requests#1165]
**Feature description**
A fun and easy item to make, it will work like crate, but will be metal, and will have some space in it to fit some items.

**Feature purpose**
A really cool thing to make as i said before, a cool addition to a lot of contraptions 

**Additional context**
it may probably have textures simmiliar to 1000kg weight or a rod.

**Comment by alikeobservant** [comment]
Seems like something the player should make.

**Comment by YourLocalPlonker** [comment]
Yeah I think if you stick 4 metal rods together you can make a cage yourself tbh
